src/Apps/CollectorProfile/Routes/Saves2/Components/SavesArtworksGrid.tsx

```diff
@@ -0,0 +1,218 @@
+import { createFragmentContainer, graphql, RelayRefetchProp } from "react-relay"
+import { SavesArtworksGrid_artworks$data } from "__generated__/SavesArtworksGrid_artworks.graphql"
+import { SavesArtworksGrid_collection$data } from "__generated__/SavesArtworksGrid_collection.graphql"
+import { useTracking } from "react-tracking"
+import ArtworkGrid from "Components/ArtworkGrid"
+import { PaginationFragmentContainer as Pagination } from "Components/Pagination"
+import {
+  ContextModule,
+  clickedMainArtworkGrid,
+  ClickedChangePage,
+  ActionType,
+  commercialFilterParamsChanged,
+} from "@artsy/cohesion"
+import { useAnalyticsContext } from "System/Analytics/AnalyticsContext"
+import { LoadingArea } from "Components/LoadingArea"
+import { useArtworkFilterContext } from "Components/ArtworkFilter/ArtworkFilterContext"
+import { FC, useState } from "react"
+import { SavesArtworksGridHeader } from "./SavesArtworksGridHeader"
+import { Spacer } from "@artsy/palette"
+import useDeepCompareEffect from "use-deep-compare-effect"
+import { usePrevious } from "Utils/Hooks/usePrevious"
+import { isEqual } from "lodash"
+import { Jump } from "Utils/Hooks/useJump"
+import { allowedFilters } from "Components/ArtworkFilter/Utils/allowedFilters"
+import { SavesEmptyStateFragmentContainer } from "./SavesEmptyState"
+
+interface SavesArtworksGridProps {
+  artworks: SavesArtworksGrid_artworks$data
+  collection: SavesArtworksGrid_collection$data
+  collectionID: string
+  relayRefetch: RelayRefetchProp["refetch"]
+}
+
+/**
+ * In the future we plan to use the `BaseArtworkFilter` and `ArtworkFilterArtworkGrid` components
+ * when filter support is added.
+ */
+const SavesArtworksGrid: FC<SavesArtworksGridProps> = ({
+  artworks,
+  collection,
+  relayRefetch,
+}) => {
+  const { trackEvent } = useTracking()
+  const {
+    contextPageOwnerType,
+    contextPageOwnerSlug,
+    contextPageOwnerId,
+  } = useAnalyticsContext()
```

Comment:
**questions (non-blocking):**
- Do we need to wrap this app in a context for this to work?
- Confirming that we don't have an analytics schema yet, correct?

Reply:
> Confirming that we don't have an analytics schema yet

I think so. but I left [TODO comment](https://github.com/artsy/force/blob/c5c20a9f3092ffbc376ffd224b8c87b32329bf7b/src/Apps/CollectorProfile/Routes/Saves2/Components/SavesArtworksGrid.tsx#L164) to clarify this point later

Reply:
Added some changes: https://github.com/artsy/force/pull/11865/commits/77dd1d15adafdade1b355788c2789ca0aad2201d